orchestra/static/orchestra/timing/time-input/time-input.directive.es6.js

```diff
@@ -0,0 +1,38 @@
+import template from './time-input.html'
+import './time-input.scss'
+// import moment from 'moment-timezone'
+
+export default function timeInput () {
+  return {
+    template,
+    scope: {
+      datetime: '='
+    },
+    link: (scope, elem, attrs) => {
+      if (scope.datetime) {
+        scope.datetime.seconds(0)
+        scope.datetime.milliseconds(0)
+        scope.timeDisplay = scope.datetime.toDate()
+      } else {
+        scope.timeDisplay = null
+      }
+
+      scope.onChange = () => {
```

Comment:
if we have the onChange, do we need any of the `watch`es or anything like that?

Reply:
Yes. The listener on watch automatically set time to default when the user selects a date from date picker. onChange only knows about the local changes made by the time-input directive.